frontend/libs/erxes-ui/src/modules/blocks/components/Editor.tsx

```
@@ -1,62 +1,55 @@
   });
 
+  useEffect(() => {
+    async function loadInitialHTML(initialHTML?: string) {
+      if (!initialHTML) return;
+      const blocks = await editor.tryParseHTMLToBlocks(initialHTML);
+      editor.replaceBlocks(editor.document, blocks);
+    }
+    if (isHTML) {
+      loadInitialHTML(initialContent);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

ESLint

[error] Definition for rule 'react-hooks/exhaustive-deps' was not found.

(react-hooks/exhaustive-deps)